Makefile

```diff
@@ -695,39 +695,13 @@ package-capi:
 		cp target/$(HOST_TARGET)/release/libwasmer.a package/lib/libwasmer.a ;\
 	fi
 
-package-capi-headless: build-capi-headless
```

Comment:
Why did we removed this?

Reply:
This step is already included in `package-capi`, this step is useless.

However, if `package-capi-headless` runs after `package-capi`, it can happen that `$(TARGET_DIR)` is set to `target/release`, so it happened that it copied `target/release/libwasmer.a` to `libwasmer-headless`, not `target/headless/.../libwasmer.a`. So you suddenly get a `libwasmer-headless.a` that is a duplicate of `libwasmer.a`.

It's better to just have one `package-capi` step that also accounts for the `-headless` stuff.

Reply:
I added it back, but removed all of the steps for `package-capi-headless`, since they are now part of the `package-capi` step: https://github.com/wasmerio/wasmer/pull/3558/commits/a23c02f927470d2dda3106d12e551de086af5536